webpack.config.js

```diff
@@ -79,20 +79,15 @@ module.exports = function makeConfig(grunt) {
                 // ensure that react is loaded only once (3rd party apps can load more...)
                 react: path.resolve('./node_modules/react'),
             },
-            extensions: ['.js', '.jsx'],
+            extensions: ['.js', '.ts', '.tsx'],
```

Comment:
I think we might need to keep jsx working for superdesk-planning

Reply:
Guys be lenient allow both. Are we planning to migrate angular to 2+?

Reply:
I forgot the apps are using the same config, will fix this. @mdhaman there are no plans for angular 2+ as far as I know.